opent2t-reference/Opent2tHelper.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+var q = require('q');
```

Comment:
Could you file a git issue/tfs track to track this? thanks a ton

---
In reply to: [101380971](https://github.com/openT2T/opent2t-cli/pull/16#discussion_r101380971) [](ancestors = 101380971,101373959)